index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Scout</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <div class="empty-element none"></div>
+        <div class="header-inner">
+          <a href="#" class="logo">
+            <p>Skout</p>
+          </a>
+          <nav class="container-header-menu none">
+            <ul class="header-menu">
+              <li href="#" class="active-link">WHY SKOUT</li>
+              <li href="#">PRODUCTS</li>
+              <li href="#">PARTNERS</li>
+              <li href="#">COMPANY</li>
+              <li href="#">TIPS&TOOLS</li>
+              <li href="#">BLOG</li>
+            </ul>
+
+            <div class="enter-buttons">
+              <button>Get Skout</button>
+              <button>Log In</button>
+            </div>
+          </nav>
+
+          <nav class="hamburger-container">
+            <button class="hamburger-icon" id="hamburger"></button>
+          </nav>
+        </div>
+
+        <header class="header-text"><span>Tips & Tools</span></header>
```

Comment:
you have already have `header` on the page, look at semantic meaning of tag `header`

Reply:
done